src/utils/logger.js

```diff
@@ -36,6 +36,9 @@ const Logger = {
     }
   },
   sentry(...args) {
+    if (__DEV__) {
```

Comment:
drop it?

Reply:
hm I find myself doing this often, so I wouldn't mind it staying here

Reply:
(This is already in dev)